lnwallet/channel.go

```diff
@@ -2788,7 +2788,7 @@ func (lc *LightningChannel) createCommitDiff(
 // in the state when adding a new HTLC, or nil otherwise.
 func (lc *LightningChannel) validateCommitmentSanity(theirLogCounter,
 	ourLogCounter uint64, remoteChain bool,
-	predictAdded *PaymentDescriptor) error {
+	predictAdded, predictTheirs *PaymentDescriptor) error {
```

Comment:
the godoc should be updated to describe this new argument. alternatively, seems like we can just pass it in via `predictAdded` and then add it to the appropriate log based on `remoteChain`

Reply:
Possibly we would need this extra argument, to allow the case where we are predicting adding `our` add to the `remote` commitment chain, but would need to look at the callsites for the method.

Reply:
Yes can do this, if `remoteChain` is true, add to our updates. Otherwise, add to theirs.